cli/src/command.rs

```diff
@@ -63,138 +65,152 @@ pub enum Command {
+    /// Prints out list of known RGB interfaces
     Interfaces,
-    /// Prints out list of known RGB contracts.
+    /// Prints out list of known RGB contracts
     Contracts,
 
-    /// Imports RGB data into the stash: contracts, schema, interfaces, etc.
+    /// Imports RGB data into the stash: contracts, schema, interfaces, etc
     #[display("import")]
     Import {
-        /// Use BASE64 ASCII armoring for binary data.
+        /// Use BASE64 ASCII armoring for binary data
         #[arg(short)]
         armored: bool,
 
-        /// File with RGB data. If not provided, assumes `-a` and prints out
-        /// data to STDOUT.
+        /// File with RGB data
+        ///
+        /// If not provided, assumes `-a` and prints out data to STDOUT
         file: PathBuf,
     },
 
-    /// Exports existing RGB contract.
+    /// Exports existing RGB contract
     #[display("export")]
     Export {
-        /// Use BASE64 ASCII armoring for binary data.
+        /// Use BASE64 ASCII armoring for binary data
         #[arg(short)]
         armored: bool,
 
-        /// Contract to export.
+        /// Contract to export
         contract: ContractId,
 
-        /// File with RGB data. If not provided, assumes `-a` and reads the data
-        /// from STDIN.
+        /// File with RGB data
+        ///
+        /// If not provided, assumes `-a` and reads the data from STDIN
         file: Option<PathBuf>,
     },
 
-    /// Reports information about state of a contract.
+    /// Reports information about state of a contract
     #[display("state")]
     State {
-        /// Contract identifier.
+        /// Contract identifier
         contract_id: ContractId,
-        /// Interface to interpret the state data.
+        /// Interface to interpret the state data
         iface: String,
     },
 
-    /// Issues new contract.
+    /// Issues new contract
     #[display("issue")]
     Issue {
-        /// Schema name to use for the contract.
+        /// Schema name to use for the contract
         schema: SchemaId, //String,
 
-        /// File containing contract genesis description in YAML format.
+        /// File containing contract genesis description in YAML format
         contract: PathBuf,
     },
 
-    /// Create new invoice.
+    /// Create new invoice
     #[display("invoice")]
     Invoice {
-        /// Force address-based invoice.
+        /// Force address-based invoice
         #[clap(short, long)]
         address_based: bool,
 
-        /// Contract identifier.
+        /// Contract identifier
         contract_id: ContractId,
 
-        /// Interface to interpret the state data.
+        /// Interface to interpret the state data
         iface: String,
 
-        /// Value to transfer.
+        /// Value to transfer
         value: u64,
     },
 
-    /// Create new transfer.
+    /// Transfer RGB assets
     #[display("transfer")]
     Transfer {
+        /// Encode PSBT as V2
+        #[clap(short = '2')]
+        v2: bool,
```

Comment:
Should there be a default value for this?

Reply:
it is a flag, thus "default value" is `false` assigned when the flag is absent